js/client/modules/@arangodb/testutils/instance.js

```diff
@@ -509,24 +509,29 @@ class instance {
   // //////////////////////////////////////////////////////////////////////////////
 
   readAssertLogLines () {
-    const buf = fs.readBuffer(this.logFile);
-    let lineStart = 0;
-    let maxBuffer = buf.length;
+    try {
+      const buf = fs.readBuffer(this.logFile);
+      let lineStart = 0;
+      let maxBuffer = buf.length;
 
-    for (let j = 0; j < maxBuffer; j++) {
-      if (buf[j] === 10) { // \n
-        const line = buf.asciiSlice(lineStart, j);
-        lineStart = j + 1;
-        
-        // scan for asserts from the crash dumper
-        if (line.search('{crash}') !== -1) {
-          if (!IS_A_TTY) {
-            // else the server has already printed these:
-            print("ERROR: " + line);
+      for (let j = 0; j < maxBuffer; j++) {
+        if (buf[j] === 10) { // \n
```

Comment:
Could this be better handled than hard-coding the ASCII character value for LF?

Reply:
using \n literally would not work with v8-buffers.